specifyweb/frontend/js_src/lib/components/QueryBuilder/ResultsTable.tsx

```diff
@@ -169,16 +174,44 @@ function Row({
             className={`${getCellClassName(condenseQueryResults)} sticky`}
             role="cell"
           >
-            <Link.NewTab
-              className="print:hidden"
-              href={viewUrl}
-              rel="noreferrer"
-            />
+            {splitIds === undefined ? (
+              <Link.NewTab
+                className="print:hidden"
+                href={viewUrl}
+                rel="noreferrer"
+              />
+            ) : (
+              <Button.Icon
+                icon="viewList"
+                title={queryText.viewListOfIds()}
+                onClick={handleToggle}
+              />
+            )}
+            {isIdListOpen && splitIds !== undefined ? (
+              <Dialog
```

Comment:
wouldn't this be better implemented by using RecordSelectorFromIds?
I think that might be more user friendly (will let them preview the records without opening a new tab, and go between records, and easily see total count of records, and easy jump to first/last record)

plus, if you have many records, this dialog would become unusable/not user friendly, but a RecordSelectorFromIds dialog would work fine

Reply:
and then could you rename this feature from "list of ids" to something more user friendly like "list of records", or "record collection" or etc?

Reply:
It would complicate things a lot, as we need to handle the fact that a user can delete or modify a resource in the preview

Reply:
I pushed a code using `RecordSelectorFromIds`, but I actually prefer the previous code that used `Dialog`. It's simpler, easier to manage, and less prone to bugs. While the list can be long, the user can scroll through it. However, I do acknowledge that for the user, it might be easier to slide between records rather than opening them one at a time.

Reply:
I tested the dialog box solution out with a row that had about 100 record IDs, and it was simple enough to scroll through the list.  I am kind of drawn to the simplicity of the dialog solution.  @grantfitzsimmons what do you think?

Reply:
@maxpatiiuk's comments are exactly right. Database IDs should only be exposed to the user as an absolute last resort.

The records should use the table format to determine how they appear. Instead of `1`, it could use the table format for that record if available. If there is no formatter defined, IDs can be shown instead.

"List of ids" should change to "List of Records" :100:

It would be great to have some kind of "browse in forms" mechanism in that dialog so the user can preview them, but I think the dialog box solution is a good first step.

Reply:
> It would be great to have some kind of "browse in forms" mechanism in that dialog so the user can preview them, but I think the dialog box solution is a good first step.

I should have been more explicit - when I said "RecordSelectorFromIds", I meant "browse in forms" as "RecordSelectorFromIds" is how that feature is called in the code.
Implementing "browse in forms" here would be quite simple as that's exactly the use case RecordSelectorFromIds was created for